changelog/bug-1855653.md

```diff
@@ -0,0 +1,5 @@
+audience: general
+level: minor
```

Comment:
There's an argument to make this a `major` change due to the fact that it requires the `generic-worker` binary to be executable by the task user, which is not a current requirement.

Reply:
Thanks to @Eijebong for bringing this up 👍🏻